modules/clusterpool/Makefile

```diff
@@ -28,6 +28,7 @@ CLUSTERPOOL_NAME ?=
 CLUSTERPOOL_SIZE ?= 1
 CLUSTERPOOL_CHECKOUT_TIMEOUT_MINUTES ?= 10
 CLUSTERPOOL_LIFETIME ?=
+CLUSTERPOOL_GROUP_NAME ?= system:mastsers
```

Comment:
This is misspelled.

Reply:
Nice catch!

@schmidtd We're both bad apparently!!

I missed this too, I even tested this, but given that system:masters requires you to have cluster access to see it - it didn't make a difference because that's group has access to the cluster with or without this entry!

Reply:
No fair commenting on a merged PR!